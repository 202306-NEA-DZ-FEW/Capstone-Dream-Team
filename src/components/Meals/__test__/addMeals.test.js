import renderer from "react-test-renderer";
import AddMeals from "../addMeals";

jest.mock("@/util/firebase", () => ({}));

it("renders correctly", () => {
    const tree = renderer.create(<AddMeals />).toJSON();
    expect(tree).toMatchSnapshot();
});
