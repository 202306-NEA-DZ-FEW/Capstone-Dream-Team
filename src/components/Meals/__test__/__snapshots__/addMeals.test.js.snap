// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div>
    <div
      className="sm:flex sm:items-center"
    >
      <div
        className="sm:flex-auto pl-2"
      >
        <h1
          className="tracking-wider font-light font-roboto"
        >
          mealsPage.meals.title
        </h1>
        <p
          className="text-sm font-semibold text-gray-700"
        >
          mealsPage.meals.description
        </p>
      </div>
      <div
        className="mt-4 sm:mt-0 sm:ml-16 sm:flex-none"
      >
        <button
          className="inline-flex items-center justify-center rounded-md border border-transparent bg-blue-500 px-4 py-2 text-sm font-medium text-white shadow-sm hover:bg-blue-700 focus:outline-none focus:ring-2 focus:ring-blue-500 focus:ring-offset-2 sm:w-auto"
          onClick={[Function]}
        >
          mealsPage.mealForm.addMeal
        </button>
      </div>
    </div>
    <div>
      <div
        className="overflow-x-auto overflow-y-hidden rounded-md border border-gray-200 shadow-md mt-5"
      >
        <table
          className="w-full text-center text-sm rtl:text-right"
        >
          <thead
            className="text-xs  uppercase bg-blue-300 bg-opacity-50 "
          >
            <tr>
              <th
                className="px-6 py-3"
                scope="col"
              >
                mealsPage.mealsTable.image
              </th>
              <th
                className="px-6 py-3"
                scope="col"
              >
                mealsPage.mealsTable.name
              </th>
              <th
                className="px-6 py-3"
                scope="col"
              >
                mealsPage.mealsTable.quantity
              </th>
              <th
                className="px-6 py-3"
                scope="col"
              >
                mealsPage.mealsTable.price
              </th>
              <th
                className="px-6 py-3"
                scope="col"
              >
                 
              </th>
            </tr>
          </thead>
          <tbody
            className=""
          />
        </table>
      </div>
    </div>
  </div>
</div>
`;
